lib/tlog/conf_validate.c.m4

```diff
@@ -160,12 +160,12 @@ m4_define(
            `                                name, json_type_to_name(type));',
            `                return TLOG_RC_FAILURE;',
            `            }',
-           `            int i;',
-           `            for (i = 0; i < json_object_array_length(obj); i++) {',
+           `            size_t i;',
+           `            for (i = 0; i < (size_t)json_object_array_length(obj); i++) {',
            `                struct json_object *item_obj = json_object_array_get_idx(obj, i);',
            `                enum json_type item_type = json_object_get_type(item_obj);',
            `                if (item_type != json_type_string) {',
-           `                    tlog_errs_pushf(perrs, "Invalid \"%s\" item #%d type: %s",',
+           `                    tlog_errs_pushf(perrs, "Invalid \"%s\" item #%ld type: %s",',
```

Comment:
The format specifier for size_t should be `%zu` to match size_t better. See printf(3).